crates/router/src/core/payments/operations/payments_incremental_authorization.rs

```diff
@@ -106,6 +100,12 @@ impl<F: Send + Clone>
             .await
             .to_not_found_response(errors::ApiErrorResponse::PaymentNotFound)?;
 
+        if payment_attempt.get_total_amount() > request.amount {
```

Comment:
This can't be true with `get_total_amount`,  because `request.amount` is a `order_amount` which can be lower than the get_total_amount. Or In incremental_authotization they can pass total amount in request?

If genuine cases also this can throw errors.

Reply:
In incremental_authotization, request.amount is total_amount which is on top of net_amount (not order_amount here)

Reply:
Add comments explain the case